library/kong

```diff
@@ -2,12 +2,13 @@ Maintainers: Mashape Docker Maintainers <support@konghq.com> (@thekonginc)
 GitRepo: https://github.com/Kong/docker-kong.git
 # needs "Constraints" to match "centos" (which this image is "FROM")
 
-Tags: 0.12.0rc2-alpine
-GitCommit: 9f2d6a104dbf8b38cb8188429c17daef72f24196
+Tags: 0.12-alpine, 0.12.0-alpine
+GitCommit: 9a20698ccb907b1a9d473e2f41e7ccdbd7de8c99
 Directory: alpine
 
-Tags: 0.12.0rc2
-GitCommit: 9f2d6a104dbf8b38cb8188429c17daef72f24196
+Tags: 0.12-centos, 0.12.0-centos, 0.12, 0.12.0, latest
```

Comment:
This new `latest` is conflicting with the old `latest` on `0.11.2`.  Probably just drop latest from `0.11`?

Any particular reason to keep the old version listed below?  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
@yosifkit, removed the extra `latest` tag - my apologies. On removing old versions, I will discuss it with other folks before removing - thanks for the feedback.